src/app/events/event-active-registration/event-active-registration.component.html

```
@@ -68,6 +68,26 @@
           />
         </div>
       }
+      @if (cancellationCopy(registration); as cancellation) {
+        <div class="flex flex-col items-start gap-2 lg:col-span-2">
+          <p class="body-small text-on-surface-variant">
+            {{ cancellation.helperText }}
+          </p>
+          <button
+            mat-stroked-button
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)